abi-bindings/README.md

```diff
@@ -0,0 +1 @@
+This directory contains ABI bindings for the Solidity contracts in the `contracts/src/CrossChainApplications` and `contracts/src/Teleporter` directories. The files with the same name as the Solidity source files are automatically generated by the `scripts/abi_bindings.sh` script. Other files in this directory (such as the packing utilities) are manually created and maintained.
```

Comment:
nit: can we add a top level heading for `README`s

Reply:
Added